CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
```

Comment:
this is BETA.2
```suggestion
# Version 3.2.0-BETA.2
```

Reply:
yeah, we didn't release beta.2.  @kryalama  please reuse this draft: https://github.com/microsoft/ApplicationInsights-Java/releases/tag/untagged-969a656e91fa14295250